array/slidingWindow.test.ts

```diff
@@ -0,0 +1,56 @@
+/* eslint-env jest */
+// @ts-ignore ambiguous import
+import slidingWindow from "./slidingWindow.ts";
+
+describe("slidingWindow", () => {
+  it("should return list, composed of 2-tuples of consecutive elements", () => {
```

Comment:
```suggestion
  it("should an array of pairs of consecutive elements for sliding window length of 2", () => {
```

Please apply a similar fix for other cases (pairs, triplets, quartets, etc.).